src/algebra/big_operators.lean

```diff
@@ -27,24 +27,36 @@ theorem finset.exists_le {α : Type u} [nonempty α] [directed_order α] (s : fi
 directed.finset_le (by apply_instance) directed_order.directed s
 
 namespace finset
-variables {s s₁ s₂ : finset α} {a : α} {f g : α → β}
 
-/-- `s.prod f` is the product of `f x` as `x` ranges over the elements of the finite set `s`. -/
-@[to_additive "`s.sum f` is the sum of `f x` as `x` ranges over the elements
+/-- `∏ x in s, f x` is the product of `f x` as `x` ranges over the elements of the finite set `s`. -/
+@[to_additive "`∑ x in s, f` is the sum of `f x` as `x` ranges over the elements
 of the finite set `s`."]
 protected def prod [comm_monoid β] (s : finset α) (f : α → β) : β := (s.1.map f).prod
 
+end finset
+
+localized "notation `∑` binders `, ` r:(scoped f, finset.sum finset.univ f) := r" in big_operators
+localized "notation `∏` binders `, ` r:(scoped f, finset.prod finset.univ f) := r" in big_operators
+
+localized "notation `∑` binders ` in ` s `, ` r:(scoped f, finset.sum s f) := r" in big_operators
+localized "notation `∏` binders ` in ` s `, ` r:(scoped f, finset.prod s f) := r" in big_operators
```

Comment:
```suggestion
localized "notation `∑` binders `, ` r:(scoped:65 f, finset.sum finset.univ f) := r" in big_operators
localized "notation `∏` binders `, ` r:(scoped:70 f, finset.prod finset.univ f) := r" in big_operators

localized "notation `∑` binders ` in ` s `, ` r:(scoped:65 f, finset.sum s f) := r" in big_operators
localized "notation `∏` binders ` in ` s `, ` r:(scoped:70 f, finset.prod s f) := r" in big_operators
```

Then `∑` and `∏` have the same precedence as `+` and `*` and you need fewer parens.

Reply:
Thanks! That's helpful.

Reply:
This breaks a lot.  I think we need to use 64 and 69 instead, then the following works as expected:
```lean
∑ x, x + x  =  ∑ x, (x + x)
∑ x, x - x  =  ∑ x, (x - x)
∏ x, x * x  =  ∏ x, (x * x)
Π x, x + x  =  (∏ x, x) + x
```

Reply:
The last one is not what I would expect. So maybe `63` and `64`?

Reply:
Whatever they end up being, I suggest adding some comments to document the magic numbers.

Reply:
I agree with Gabriel. I would expect `∏ x, x + x  =  (∏ x, x) + x`

Reply:
I am paranoid and would put brackets whenever I detected a possible ambiguity, because if there is not 100% concensus on something like this (and there doesn't seem to be) then I would feel that any attempt to drop brackets risked confusion. On the other hand I am a number theorist who does not use `nat` or `\subset` in their papers because they also mean different things to different people and are hence, as far as I am concerned, ambiguous. In short I don't mind because I am likely to add too many brackets anyway -- I don't play golf.